views/layouts/partials/header.tpl.php

```diff
@@ -1 +1,8 @@
-<div>Header</div>
\ No newline at end of file
+<div>
+    <div>
+        <a href="/">Logo</a>
```

Comment:
replace Logo with an actual image, that should represent Website logo.
Image should be placed somewhere in our project, ideally under /images folder.
Ideally an image with transparency (.png). I attached one in jira, as example.